core/api/type/type_proto/type.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/google/keytransparency/core/api/type/type_proto";
```

Comment:
I wonder if 'types' would be a more pragmatic choice than 'type' given that 'type' is a Go reserved word.
I know the import will be type_proto but it still might be better to avoid potential confusion?

Reply:
I think this would produce a slightly odd URL for proto type definitions:
 `keytransparency/types/typename`

Reply:
OK.